docs/HOWTO_DEVELOPMENT_LOCAL_SETUP.md

```diff
@@ -0,0 +1,81 @@
+## Getting started
+
+The following is the reccomended developer set-up.  If you prefer, [click here for instructions for a minimalist set-up.](HOWTO_MINIMALIST_DEPLOY.md)
```

Comment:
This minimalist set-up is a deployment method, so it's not really a replacement for setting up a development environment. I think we should link to it in the "additional ways to deploy" in the README

Reply:
Deleted the paragraph here. Fixed in readme in commit a5ff9d6.